src/SOFe/InfoAPI/Ast/TextSegment.php

```diff
@@ -24,6 +24,7 @@
 
 /**
  * A segment of (escapes-resolved) plain-text.
+ * (The segment that InfoAPI ignore.)
```

Comment:
InfoAPI doesn't ignore it; InfoAPI still has to include this text.
```suggestion
 * A segment of (escapes-resolved) plain-text that gets appended to the output directly.
```